app/helpers/application_helper.rb

```diff
@@ -2,7 +2,7 @@ module ApplicationHelper
   include NavigationHelper
 
   def format_currency(amount, **options)
-    number_to_currency(amount, options)&.sub(/\.00$/, '')
+    number_to_currency(amount, options)
```

Comment:
We might as well remove `format_currency` completely as all it now does is proxy to `number_to_currency`. We could just use `number_to_currency` in our views and keep things closer to the Rails norm.